src/components/pages/text-curation/edit/table/form/scientific-name.tsx

```diff
@@ -88,33 +107,32 @@ export default function ScientificNameEdit({ row }) {
       <Box mb={4}>
         <HStack spacing={4}>
           {breadCrumbs.length > 0 ? (
-            <Tag>{`taxon id : ${breadCrumbs[breadCrumbs.length - 1].id}`}</Tag>
+            <Tag>{`${t("text-curation:edit.scientific_name.taxon_id")} : ${
+              breadCrumbs[breadCrumbs.length - 1].id
+            }`}</Tag>
           ) : (
-            row.taxonId && <Tag>{`taxon id : ${row.taxonId}`}</Tag>
+            row.taxonId && (
+              <Tag>{`${t("text-curation:edit.scientific_name.taxon_id")} : ${row.taxonId}`}</Tag>
+            )
           )}
 
           {breadCrumbs.length > 0 ? (
-            <Tag>{`rank : ${breadCrumbs[breadCrumbs.length - 1].rankName}`}</Tag>
+            <Tag>{`${t("text-curation:edit.scientific_name.rank")} : ${
+              breadCrumbs[breadCrumbs.length - 1].rankName
+            }`}</Tag>
           ) : (
-            row.rank && <Tag>{`rank : ${row.rank}`}</Tag>
+            row.rank && <Tag>{`${t("text-curation:edit.scientific_name.rank")} : ${row.rank}`}</Tag>
           )}
         </HStack>
       </Box>
-      {row.taxonomyMatchedNames.map((suggestion) => (
-        <Button
-          variant="outline"
-          size="xs"
-          bg="blue.50"
-          key={suggestion.taxonId}
-          colorScheme="blue"
-          borderRadius="3xl"
-          onClick={() => onTagSelect(suggestion)}
-          mb={2}
-          mr={2}
-        >
-          {suggestion.fullName}
-        </Button>
-      ))}
+
+      <RadioInputField
+        name="validatedStatus"
+        label={t("text-curation:edit.scientific_name.validity")}
+        options={validityOptions.map((v) => ({ label: v, value: v }))}
```

Comment:
`validityOptions.map((v) => ({ label: v, value: v }))` please precompute them outside of component declaration so this won't recompute this everytime

Reply:
done